src/sentry/api/endpoints/organization_sentry_function.py

```diff
@@ -1,22 +1,22 @@
 from uuid import uuid4
 
-from django.utils.text import slugify
 from rest_framework import serializers
 from rest_framework.response import Response
 
 from sentry import features
 from sentry.api.api_publish_status import ApiPublishStatus
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases import OrganizationEndpoint
+from sentry.api.helpers.slugs import sentry_slugify
 from sentry.api.serializers import serialize
 from sentry.api.serializers.rest_framework import CamelSnakeSerializer
 from sentry.models.sentryfunction import SentryFunction
 from sentry.utils.cloudfunctions import create_function
 
 
 class EnvVariableSerializer(CamelSnakeSerializer):
-    value = serializers.CharField(required=True)
-    name = serializers.CharField(required=True)
+    value = serializers.CharField(required=True, allow_blank=False, allow_null=False)
+    name = serializers.CharField(required=True, allow_blank=False, allow_null=False)
```

Comment:
Instead of manually checking in the validate function [here](https://github.com/getsentry/sentry/pull/59669/files#diff-148f76e7c3becb55218825252f4a6a5d3fb4574c410c517052fcecf80e9d5eb6L33-L34) if value and name are non-null and non-empty, we can directly check for those in the serializer fields.